content/pro/openbb-copilot.md

```diff
@@ -79,57 +94,70 @@ the widget has been added to the dashboard) to provide an answer.
-|:-----------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------:|
+Once uploaded, OpenBB Copilot can use the data from these files to answer your questions. If it does, it will cite the specific files (and, for PDFs, the exact page) it referenced.
 
-| ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/table_widget_creation_button.jpg) | ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/table_widget_creation.jpg) | ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/table_widget_created.jpg) |
-|:-----------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------:|
+:::note
 
-## Querying specific widgets only
+OpenBB Copilot uses filenames to determine if a file is relevant to your query. For best results, use descriptive filenames.
+For example, if you have a technical report from Tesla released in 2024, a good filename would be `tsla_technical_report_2024.pdf`.
 
-Sometimes, you might want OpenBB Copilot to analyze only specific widgets on your dashboard. For example, if you want to dive deep into an earnings transcript without considering other data, you can do that.
+:::
 
-To focus on certain widgets, click the "Add widgets as context" button on the widgets you want OpenBB Copilot to use. Once selected, OpenBB Copilot will only access data from those widgets, ignoring all others. This allows you to carry out a more targeted analysis while using OpenBB Copilot as usual.
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/add_custom_data_chat_example.jpg" alt="add custom data openbb copilot" width="80%" height="80%" />
 
-<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/add_specific_widget_context.jpg" alt="add widget context openbb copilot" width="80%" height="80%" />
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/custom_data_chat_example.jpg" alt="query custom data openbb copilot" width="80%" height="80%" />
 
-<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/aapl_earnings_transcript_chat_example.jpg" alt="query widget context openbb copilot" width="80%" height="80%" />
+### Adding custom data as a widget
```

Comment:
In this section I'd not go into details on how to add a widget from a file, backend or wtv. It duplicates documentation and leaves space for inconsistent or outdated content.

Instead I suggest adding a info note linking to the relevant sections that explain in detail how to do this.